package.json

```diff
@@ -14,7 +14,7 @@
     "ethers": "^4.0.27",
     "group-array": "^0.3.3",
     "kosmos-schemas": "^1.1.2",
-    "kredits-contracts": "^5.1.1",
+    "kredits-contracts": "github:67P/kredits-contracts#feature/gitea_site",
```

Comment:
let's first release a new kredits-contract version

Reply:
PR is open over there: https://github.com/67P/kredits-contracts/pull/113